vllm/v1/core/scheduler.py

```diff
@@ -61,36 +66,62 @@ def __init__(
         # Request id -> RunningRequestData
         self.running_reqs_data: Dict[str, RunningRequestData] = {}
 
-    def schedule(self) -> "SchedulerOutput":
-        scheduled_new_reqs: List[Request] = []
-        scheduled_resumed_reqs: List[Request] = []
-        scheduled_running_reqs: List[Request] = []
-        preempted_reqs: List[Request] = []
+        # Encoder-related.
+        # NOTE(woosuk): Here, "encoder" includes the vision encoder (and
+        # projector if needed). Currently, we assume that the encoder also
+        # has the Transformer architecture (e.g., ViT).
+        # FIXME(woosuk): Below are placeholder values. We need to calculate the
+        # actual values from the configurations.
+        self.max_num_encoder_input_tokens = 2048
+        self.encoder_cache_manager = EncoderCacheManager(cache_size=2048)
```

Comment:
This means we always initialize an encoder cache regardless of the model type (text-only or multimodal).

I don't think this is an issue technically because for text-only models there's no multimodal data during profiling (and thus no memory usage for encode cache), but we should probably make a note here to indicate so, otherwise it might be misleading that we're reserving space for text-only models.